HomePage.java

```diff
@@ -1 +1,3 @@
-// created homepage with React and Bootstrap
\ No newline at end of file
+// created homepage with React and Bootstrap
+
+// Handled bug
```

Comment:
please explain it clearly

Reply:
i handled the bug in home page now it working